docs/development/docs.rst

```diff
@@ -10,27 +10,50 @@ The docs are kept in the ``docs/`` directory at the top of the source tree.
    However, if there is going to be periodic/bigger contributions, it may be a
    good idea to install the Sphinx requirements to build our docs.
 
+Contributing through the Github UI
+----------------------------------
+
+If you're making small changes to the documentation, you can verify those changes through the documentation generated when you open a PR and can be accessed using the Github UI.
+
+1. click the checkmark next to your commit and it will expand to have multiple options
+2. right-click the "details" link next to the "docs/readthedocs.org:docs" item
+3. navigate to the section of the documentation you worked on to verify your changes
+
+Contributing from your local machine
+------------------------------------
+
+If you're making large changes to the documentation, you'll want to verify those changes locally before pushing upstream.
+
+1. follow the instructions to `Set up your environment <https://docs.readthedocs.io/en/stable/development/install.html#set-up-your-environment>`_
```

Comment:
When linking to internal pages we use the `:ref:` or `:doc:` roles https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations. You can see an example here https://github.com/readthedocs/readthedocs.org/blob/8009046a9c416816ddc3448c8427d9c683f0e65d/docs/config-file/v2.rst#L259-L262

But looks like you are going to change this anyway :D